mwviews/api/pageviews.py

```diff
@@ -301,6 +303,120 @@ def top_articles(
             'The pageview API returned nothing useful at: {}'.format(url)
         )
 
+    def top_articles_by_country(
+            self, project, access='all-access',
+            year=None, month=None, limit=1000):
+        """
+        Get pageviews to a project, split by country of origin for a given month
```

Comment:
Maybe to make it a little clearer:

Get a list of countries that access this project, sorted from highest to lowest pageview traffic, for a given month

Reply:
Great -- updated (though I took out "list of" so it wouldn't exceed Python's suggested line length)